ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java

```diff
@@ -452,7 +453,7 @@ private static ImportTableDesc getBaseCreateTableDescFromTable(String dbName,
     boolean isAutoPurge = false;
     boolean needRecycle = false;
     boolean copyToMigratedTxnTable = replicationSpec.isMigratingToTxnTable();
-
+    boolean performOnlyMove = replicationSpec.isInReplicationScope() && Utils.onSameHDFSFileSystem(dataPath, tgtPath);
```

Comment:
why do we need this extra condition replicationSpec.isInReplicationScope()?

Reply:
As we would like to use it only in replication cases.